source/includes/administration/_quotas.md

```diff
@@ -0,0 +1,68 @@
+## Quotas
+
+Quotas are used to limit the resource usage on a service connection consumed by an organization. The primary use case for CloudMC is to enforce limited quotas on trial accounts. Quotas can target different resources. For example, number of vCPUs, allocated memory, disk size, network bandwidth, number of public IPs and more.
+
+<!------------------- LIST QUOTAS --------------------->
```

Comment:
How do you want to handle the documentation of other operations? In the same PR or we will updated this document as we go along?

I am asking because the PR is pointing at `cmc-dev`.

Reply:
We usually open a separate PR for every story.

Reply:
I see, thanks. When are the docs published to https://cloudops.github.io/cloudmc-api-docs/#getting-started? If it's when we merge to `cmc-dev`, we might end-up publish docs for an API that is not yet available.

Reply:
As discussed, I will split this PR in two, one for adding the document and another only for the create operation.